<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QMainWindow" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>585</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>585</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(247, 252, 255);
color:rgb(1, 31, 54);\n</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="3,2,1,1,1,1">
    <property name="leftMargin">
     <number>25</number>
    </property>
    <property name="topMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>25</number>
    </property>
    <property name="bottomMargin">
     <number>15</number>
    </property>
    <item>
     <widget class="QLabel" name="label_login">
      <property name="font">
       <font>
        <family>Fira Sans</family>
        <pointsize>28</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:28pt;
background:rgba(248, 253, 255, 206);
color:rgb(1, 31, 54);</string>
      </property>
      <property name="text">
       <string>Login to Linkedin</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>15</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_email">
        <property name="styleSheet">
         <string notr="true">font-size:15pt ;color:rgb(52, 52, 52)</string>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_password">
        <property name="styleSheet">
         <string notr="true">font-size:15pt ;color:rgb(52, 52, 52
)</string>
        </property>
        <property name="text">
         <string>password</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="password_lineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>38</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:14pt;color:rgb(52,52 , 52)</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="email_lineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>38</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:14pt;color:rgb(52,52 , 52)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>17</width>
        <height>53</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="login_pushButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
color:white;
border:1px solid white;
border-radius:5px;
padding:5px;
background:rgb(68, 170, 248)
}
QPushButton:hover
{
	border:1px solid rgb(32, 74, 135);
	border-radius:5px;
	background:rgb(48, 162, 249);
}</string>
      </property>
      <property name="text">
       <string>login</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>17</width>
        <height>52</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_dont_have_acc">
        <property name="styleSheet">
         <string notr="true">color: rgb(52, 52, 52);</string>
        </property>
        <property name="text">
         <string>Don't have an account?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="signup_pushButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
color:white;
border:1px solid white;
border-radius:5px;
padding:5px;
background:rgb(68, 170, 248)
}
QPushButton:hover
{
	border:1px solid rgb(32, 74, 135);
	border-radius:5px;
	background:rgb(48, 162, 249);
}</string>
        </property>
        <property name="text">
         <string>Sign up</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>585</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
